tests/test_block.py

```diff
@@ -0,0 +1,63 @@
+from unittest.mock import MagicMock, patch
```

Comment:
i understand that this allows to static testing, is there a way of providing also prod tests like the rest?

Reply:
afaik the others aren't prod tests - they are assuming the block was correctly decoded and operating on the json